elements/mlnx-ofed/install.d/92-install-ofed-packages

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-1} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+yum install -y ${DIB_MLNX_OFED_PKGLIST:-mlnx-ofed-hpc mlnx-fw-updater}
```

Comment:
Doesn't need to be changed, but for next time, a file in environment.d can be used to set variables and their defaults.

https://docs.openstack.org/developer/diskimage-builder/developer/developing_elements.html#environment-variables